src/screens/ElectionsScreen.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
```

Comment:
I don't think this file is supposed to be here anymore

Reply:
Yes I think all screens are now grouped in folders